src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -208,4 +329,12 @@ class JooqTouchstoneRepository(
             record[COVERAGE.TARGET],
             record[COVERAGE.COVERAGE_]
     )
+
+    fun mapDemographicRow(record: Record) = DemographicRow(
+            record[DEMOGRAPHIC_STATISTIC.COUNTRY],
+            "${record[DEMOGRAPHIC_STATISTIC.AGE_FROM]} - ${record[DEMOGRAPHIC_STATISTIC.AGE_TO]}",
+            record[DEMOGRAPHIC_STATISTIC.YEAR],
+            record[GENDER.NAME],
+            record[DEMOGRAPHIC_STATISTIC.VALUE]
+    )
```

Comment:
The main query to retrieve the rows is written in the most naive way possible, as I wanted to be really clear I had got the logic right first. I think it can and should be optimized though, so lets open a ticket for that.